src/mds/MDCache.cc

```diff
@@ -448,13 +448,6 @@ void MDCache::create_mydir_hierarchy(MDSGather *gather)
     stray->store_backtrace(gather->new_sub());
   }
 
-  CInode *journal = create_system_inode(MDS_INO_LOG_OFFSET + mds->whoami, S_IFREG);
-  string name = "journal";
-  CDentry *jdn = mydir->add_primary_dentry(name, journal);
-  jdn->_mark_dirty(mds->mdlog->get_current_segment());
-
-  mydir->fnode.fragstat.nfiles++;
-  mydir->fnode.rstat.rfiles++;
```

Comment:
I thought we had a few places (in vestigial shutdown code or whatever)
where we were trying to verify that we had cleaned stuff up by checking
that the number of recursive files was less than 3 (or whatever). I can't
find any now so I'm probably mixing it up with other things. :)